lib/http3/server.c

```diff
@@ -1473,6 +1476,8 @@ h2o_http3_conn_t *h2o_http3_server_accept(h2o_http3_server_ctx_t *ctx, quicly_ad
     req_scheduler_init(conn);
     conn->scheduler.uni.active = 0;
     conn->scheduler.uni.conn_blocked = 0;
+    memset(&conn->_conns, 0, sizeof(conn->_conns));
+    h2o_linklist_insert(&ctx->accept_ctx->ctx->http3._conns, &conn->_conns);
```

Comment:
Please use `h2o_linklist_init_anchor`.

Reply:
Do you mean this?
```
h2o_linklist_init_anchor(&conn->_conns);
```
I'm afraid it would cause an assertion failure `!h2o_linklist_is_linked(node)`  in `h2o_linklist_insert`.

Reply:
Please use `h2o_linklist_is_empty` for checking if an anchor refers to something.

Reply:
Ah my bad, `conn->_conns` is a node element, not an anchor. Yes, what you have now is fine.

Reply:
Got it. Thank you for checking.